File: we_11_1_fusing_three_independent_sensors_by_information.m
% Integrated Navigation Systems — Chapter 11: Multisensor and All-Source Fusion
% Worked Example 11.1 — Fusing three independent sensors by information
% Companion code (educational reference implementation; see the book for context).

v=[4 1 9]; info=sum(1./v); Pf=1/info;
fprintf('fused var=%.3f m^2\n', Pf);   % 0.735

File: we_11_1_fusing_three_independent_sensors_by_information.py
# Integrated Navigation Systems — Chapter 11: Multisensor and All-Source Fusion
# Worked Example 11.1 — Fusing three independent sensors by information
# Companion code (educational reference implementation; see the book for context).

v=[4,1,9]; Pf=1/sum(1/x for x in v)
print(f'fused var={Pf:.3f} m^2')   # 0.735
